Lib/functools.py

```diff
@@ -273,33 +273,100 @@ def reduce(function, sequence, initial=_initial_missing):
 ### partial() argument application
 ################################################################################
 
+
+class PlaceholderType:
+    """PlaceholderType()
+    --
+
+    The type of the Placeholder singleton.
+    Used as a placeholder for partial arguments.
+    """
```

Comment:
```suggestion
    """The type of the Placeholder singleton.
    
    Used as a placeholder for partial arguments.
    """
```

I do not think the headings is really needed. It is clear that this is class docstring (and from a Sphinx' autodoc or napoleon PoV, it makes more sense, although CPython uses neither of the two).